src/services/item/service.ts

```diff
@@ -682,8 +682,16 @@ export class ItemService {
       await this.hooks.runPreHooks('copy', actor, repositories, { original });
     }
 
+    const siblings: Item[] = [];
+    if (parentItem) {
+      siblings.push(...(await itemRepository.getChildren(parentItem)));
+    } else {
+      const itemMemberships = await itemMembershipRepository.getAccessibleItems(actor, {}, null);
+      siblings.push(...itemMemberships.data.map((m) => m.item));
+    }
+
     // TODO: args?
```

Comment:
Should we remove this todo ?

Reply:
I have no idea what it is about. I removed it !